### DISCUSSION TITLE
How to properly integrate stitches styling

### DISCUSSION THREAD

**Discussion by itsdillon** [remix-run/remix#647] [Q&A] [answered]
I'm able to successfully get stitches working by using `getCssText()` inside the remix app's root file.

I place `<style id="stitches" dangerouslySetInnerHTML={{ __html: getCssText() }} />` in the `<head>` but I get a mismatch from server to client.

Any one know how I would go about integrating stitches inside remix?

**Comment by sergiodxa** [comment]
If you check in the Discord, someone shared that you could use [resource routes](https://remix.run/docs/en/v1/guides/resource-routes) to call `getCssText()` and return the styles as a separate CSS file.

**Comment by machour** [comment, marked as answer]
There is now an example for this: https://github.com/remix-run/examples/tree/main/stitches
